userguide/virtualmachines.rst

```diff
@@ -410,19 +419,21 @@ click |ui-add|, then set the :guilabel:`Type` to :guilabel:`Raw File`.
    Raw File Disk Device
 
 
-:guilabel:`Browse` to select the image file. If a specific sector size
-is required, enter the number of bytes in :guilabel:`Disk sector size`.
-The default value of *0* uses an autotuner to find and set the best
-sector size for the file.
+Click |ui-browse| to select the image file. If a specific sector size
+is required, open the :guilabel:`Disk sector size` drop-down menu and
+pick between *512* and *4096* bytes. The *Default* value automatically
```

Comment:
Why go so specific on this?
"If a specific sector size is required, choose it from Disk sector size.
*Default* automatically selects a preferred size."